primitives/bridge/src/lib.rs

```diff
@@ -67,6 +67,18 @@ mod custom_send_message;
 pub enum Command {
     // TODO: add real commands here
     Test(Vec<u8>),
```

Comment:
Should we remove `Test`?

Reply:
I would keep it for now. I think it's still useful if we want to test message sending in a general way.

Reply:
@ParthDesai what do you think?

Reply:
Okay. Let's keep it for now.

Reply:
*fixed*